jenkins/check-for-build.jenkinsfile

```diff
@@ -89,6 +92,7 @@ pipeline {
                                 )
                             } catch (err) {
```

Comment:
Can we remove this catch block, its just swallowing errors that should be failing the build

Reply:
+1 here. I think the exceptions threw above will be caught here and won't fail the job.

Reply:
Good catch - I was adding on to existing code but see this would silently fail. Removed in latest commit.